### DISCUSSION TITLE
v0.6.3

### DISCUSSION THREAD

**Discussion by dail8859** [dail8859/NotepadNext#389] [New Releases]
## What's Changed
* Update Qt to v6.5
* Add more Book Mark features
* Add Turkish translation
* Allow escaped characters and comments in JSON
* Detect INI and JSON files based on file contents
* Fix Highlighted Scroll Bar showing smart highlight
* Fix regex search performance
* Fix Select Next action using incorrect search flags
* Fix opening multiple files at once
* Respect tab and tabwidth in language files
* Fix incorrect locale detection loading wrong language
* Fix auto-complete box popping up on Mac when typing
* Fix crash on macOS in entry of multi-character strings with IME
* Fix hex view font family issue on MacOS
* Automatically resize search results dock contents
* Collapse top level items when a new search result is added
* Fix search result font size being too small on Mac
* Fix C# highlighting

**Full Changelog**: https://github.com/dail8859/NotepadNext/compare/v0.6.1...v0.6.3

<hr /><em>This discussion was created from the release <a href='https://github.com/dail8859/NotepadNext/releases/tag/v0.6.3'>v0.6.3</a>.</em>

**Comment by ghost** [comment]
简易带点皮肤或者是颜色
